app/scripts/lib/AbstractPetnamesBridge.ts

```diff
@@ -114,14 +114,42 @@ export abstract class AbstractPetnamesBridge {
   /**
    * Update the Source with the given entry. To be overridden by two-way subclasses.
    *
-   * @param type
-   * @param entry
+   * @param _type
+   * @param _entry
    */
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  protected updateSourceEntry(type: ChangeType, entry: PetnameEntry): void {
+  protected updateSourceEntry(_type: ChangeType, _entry: PetnameEntry): void {
     throw new Error('updateSourceEntry must be overridden for two-way bridges');
   }
 
+  /**
+   * This predicate describes a subset of Petnames state that is relevant
+   * to the bridge.
+   *
+   * By default, the isSyncParticipant method returns true for all Petnames,
+   * meaning every PetnameEntry in the NameController state is considered for
+   * synchronization. This would result in Petnames state being a mirror of
+   * the source entries or vice versa after synchronization
+   *
+   * If you override this method to return false for some Petnames, those
+   * entries are effectively 'masked' or excluded from the synchronization
+   * process. This has a couple of implications:
+   *
+   * Source->Petnames direction: Masked Petname entries will not be deleted.
+   * If isSyncParticipant returns false for some target Petname,that entry
+   * will not be deleted from Petnames state during synchronization.
+   *
+   * Petnames->Source direction: Masked Petname entries will not be added to
+   * Source.  If isSyncParticipant returns false for some Petname, that entry
+   * will not be added to the Source list during synchronization.
+   *
+   * @param _targetEntry - The entry from Petname state to check for membership.
+   * @returns true iff the target Petname entry should participate in synchronization.
+   */
+  protected isSyncParticipant(_targetEntry: PetnameEntry): boolean {
```

Comment:
This is great, but I wonder if a more explicit name would be something like `shouldSyncPetname` so it's clear we're talking about the petnames only and not the source?

Reply:
Done.